packages/react-components/src/unstable/tsconfig.json

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": "../../tsconfig.json",
```

Comment:
adding a `package.json` needs another tsconfig file otherwise vscode will not be happy